static/app/views/performance/queues/messageSpanSamplesTable.spec.tsx

```diff
@@ -0,0 +1,23 @@
+import {OrganizationFixture} from 'sentry-fixture/organization';
+
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import useOrganization from 'sentry/utils/useOrganization';
+import {MessageSpanSamplesTable} from 'sentry/views/performance/queues/messageSpanSamplesTable';
+
+jest.mock('sentry/utils/useOrganization');
+
+describe('messageSpanSamplesTable', () => {
+  const organization = OrganizationFixture();
+  jest.mocked(useOrganization).mockReturnValue(organization);
+
+  beforeEach(() => {});
+  it('renders', () => {
+    render(<MessageSpanSamplesTable data={[]} isLoading={false} />);
+    screen.getByText('Span ID');
+    screen.getByText('Message ID');
+    screen.getByText('Processing Latency');
+    screen.getByText('Message Size');
+    screen.getByText('Status');
```

Comment:
If you check the HTTP module specs for inspo, there are some nice tests in there that check for a `table` role, and `cell` roles to make sure there's a table 👍🏻 Up to you, though

Reply:
Nice, I've updated the tests to check by role!